endpoints/openrtb2/auction_test.go

```diff
@@ -4650,6 +4654,69 @@ func TestValidateStoredResp(t *testing.T) {
 	}
 }
 
+func TestValidResponseWhenRequestRejected(t *testing.T) {
+	nbr := openrtb3.NoBidReason(123)
+	reject := hookexecution.RejectError{
+		int(nbr),
+		hookexecution.HookID{"foobar", "foo"},
+		hooks.StageEntrypoint.String(),
+	}
+
+	testCases := []struct {
+		description         string
+		expectedBidResponse openrtb2.BidResponse
+		hookExecutor        hookexecution.HookStageExecutor
+	}{
+		{
+			"Assert correct BidResponse when request rejected at entrypoint stage",
+			openrtb2.BidResponse{ID: "some-request-id", NBR: &nbr},
+			rejectableHookExecutor{entrypointReject: &reject},
```

Comment:
Please always include labels on object initializers. If there is only one test case, please remove the table driven structure.

Reply:
We will add new test cases in the next PRs here too.